pkg/ccl/streamingccl/streamingest/stream_ingestion_manager_test.go

```diff
@@ -46,7 +46,7 @@ func TestRevertTenantToTimestamp(t *testing.T) {
 	systemSQL.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.target_duration = '100ms'")
 	systemSQL.Exec(t, "SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval ='100ms'")
 	systemSQL.Exec(t, "SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval ='100ms'")
-	systemSQL.Exec(t, "CREATE VIRTUAL CLUSTER target")
+	systemSQL.Exec(t, "CREATE VIRTUAL CLUSTER target WITH SERVICE NONE")
```

Comment:
Can we open an issue for the problem we discovered here? I think we found some pretty suspect behavior on rapid state changes.

Reply:
https://github.com/cockroachdb/cockroach/issues/119724